third_party/ibis/ibis_oracle/datatypes.py

```diff
@@ -50,8 +50,8 @@ def _get_type(col: _FieldDescription) -> dt.DataType:
         raise NotImplementedError(f"Oracle type {typename} is not supported")
 
     if typename == cx_Oracle.DB_TYPE_NUMBER:
-        if col[4] == 0 and col[5] == -127:
-            # This will occur if type is NUMBER with no precision/scale
+        if col[5] == -127:
```

Comment:
I know this was pre-existing. Is it possible to point to some documentation about these values - ie. -127

Reply:
I don't believe it is documented anywhere. Perhaps in an Oracle support note but, to my knowledge, nowhere publicly visible.